viz.Rmd

```diff
@@ -547,16 +788,26 @@ are blending together, and although it is possible to derive *some* insight from
 of the most incorrect measurements), it isn't the clearest way to convey our message and answer the question. Let's try a different strategy of 
 creating multiple separate histograms on top of one another. 
 
-In order to create a plot in `ggplot2` that has multiple subplots arranged in a grid, we use the `facet_grid` function.
-The argument to `facet_grid` specifies the variable(s) used to split the plot into subplots. It has the syntax `vertical_variable ~ horizontal_variable`,
-where `veritcal_variable` is used to split the plot vertically, `horizontal_variable` is used to split horizontally, and `.` is used if there should be no split
-along that axis. In our case, we only want to split vertically along the `Expt` variable, so we use `Expt ~ .` as the argument to `facet_grid`.
+In order to create a plot in `ggplot2` 
+that has multiple subplots arranged in a grid, 
+we use the `facet_grid` function.
+The argument to `facet_grid` specifies the variable(s) used to split the plot 
+into subplots, and how to split them (i.e., into rows or columns).
+If the plot is to be split horizontally, into rows, 
+then the `rows` argument is used.
+If the plot is to be split vertically, into columns, 
+then the `columns` argument is used.
+Both the `rows` and `columns` argument take the column names to split the data 
+on when creating the subplots. 
+One key thing is that the column names must be surrounded by the `vars` function.
+This function allows the column names to be correctly evaluated 
```

Comment:
could we explain the `vars` function a bit more here

Reply:
We can't really without going down a deep rabbit hole about unquoted column names in R, and how to program with them... 

For example, here are the docs for `vars`:
Just like `aes()`, `vars()` is a quoting function that takes inputs to be evaluated in the context of a dataset. 

To explain that we need to explain why R is weird by using unquoted column names, and what it means to evaluate something in the context of a dataset - which means explaining the global environment... and on...